roles/openshift_prometheus/defaults/main.yaml

```diff
@@ -0,0 +1,35 @@
+---
+# defaults file for openshift_prometheus
+openshift_prometheus_state: present
+
+openshift_prometheus_namespace: prometheus
+
+openshift_prometheus_replicas: 1
+openshift_prometheus_node_selector: {"region":"infra"}
+
+# container resources
+openshift_prometheus_cpu_limit_prometheus: null
+openshift_prometheus_memory_limit_prometheus: null
```

Comment:
You could probably just drop 'prometheus' for these values assuming you make the change for better readability